src/app/app-wrapper.js

```diff
@@ -19,7 +20,9 @@ const AppWrapper = () => {
                     <QueryParamProvider ReactRouterRoute={Route}>
                         <CurrentItemProvider>
                             <RightHandPanelProvider>
-                                <App />
+                                <PrintAreaProvider>
```

Comment:
I added a new context to be able to communicate between the `options` menu where the print trigger is, and the datasets to print. The library requires a `ref` to the area to print, and we now set it in the context that is consumed and triggered from the options menu. Please let me know if there is a different approach, or if I could add the functionality to one of the already existing providers.

Reply:
I think that using a context provider is the right approach here. We've also been talking about perhaps adding global state to this app. If we did that, I expect that this part would actually not be included in the global state, since a) this provider's value is a ref and b) I don't think this provider is going to trigger a lot of re-renders. As such, I also think that keeping this is a separate provider is better than integrating it into an existing one.